src/AutorestSwift/Code Generator/jsonrpc/Model.swift

```diff
@@ -326,6 +326,60 @@ public enum RPCObject: Equatable {
             self = .dictionary(value.mapValues { RPCObject($0) })
         }
     }
+
+    public var asString: String? {
+        switch self {
+        case let .string(value):
+            return value
+        default:
+            return nil
+        }
+    }
+
+    public var asList: [RPCObject]? {
+        switch self {
+        case let .list(values):
+            return values
+        default:
+            return nil
+        }
+    }
+
+    public var asInt: Int? {
+        switch self {
+        case let .integer(value):
+            return value
+        default:
+            return nil
+        }
+    }
+
+    public var asDouble: Double? {
+        switch self {
+        case let .double(value):
+            return value
+        default:
+            return nil
+        }
+    }
+
+    public var asBool: Bool? {
+        switch self {
+        case let .bool(value):
+            return value
+        default:
+            return nil
+        }
+    }
+
+    public var asDict: [String: RPCObject]? {
+        switch self {
+        case let .dictionary(value):
+            return value
+        default:
+            return nil
+        }
+    }
```

Comment:
These needed to work with `RPCObject` in anything resembling a reasonable fashion.